src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString
+    .split(';')
+    .map((item) => item.split(':'))
+    .map((item) => item.map((value) => value.trim()))
```

Comment:
These two `map` operations can be combined into a single `map` call to improve performance and readability. You can trim and split the item in one go.